### PR TITLE
added new components and fixed issues

### PR DESCRIPTION
**Pull request by darsh399**

### CODE REVIEW

**Comment by darsh399** [comment]
@aniro45 @abhie53   Review this code

File: src/components/NavigationComponents/Contact_Us.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import Message from "./Message/Message";
+import phoneIcon from './../images/phone.png';
+import emailIcon from './../images/email.png';
+import addressIcon from './../images/location.png'
+const Contact_Us = () => {
+    return (
+        <div className="contact_page_container">
+            <div className="message-container">
+                <p className="message-text">Get In Touch</p>
+                <h1>Let's Chat, Reach Out To Us</h1>
```
Comment (aniro45) [inline src/components/NavigationComponents/Contact_Us.js]: take this message in message file

File: src/components/NavigationComponents/Home.js [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
 import './Home.css';
-
+import Message from "./Message/Message";
 const Home = () => {
     return (
         <div className="home-container">
             <div>
-                <h1>Welcome To SVY BIOTECH</h1>
+                <h1>{Message.greet}</h1>
                 <p>A Division Of Sieben Healthcare</p>
```
Comment (aniro45) [inline src/components/NavigationComponents/Home.js]: take this ass well

File: src/components/NavigationComponents/Contact_Us.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <p className="message">{Message.contactus_msg}</p>
+                <hr />
+                <div className="input-container">
+                    <div className="name-container">
+                        <label htmlFor="firstname">First Name</label>
+                        <input type="text" id="firstname" placeholder="First Name" />
+                        <label htmlFor="lastname">Last Name</label>
+                        <input type="text" id="lastname" placeholder="Last Name" />
```
Comment (aniro45) [inline src/components/NavigationComponents/Contact_Us.js]: extract this input in separate component

**Review by aniro45 — APPROVED** [review]

**Pull request merged**